addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4552,6 +4552,14 @@ export class OdooEditor extends EventTarget {
             ancestors(block, this.editable).includes(this.editable)
         ) {
             this.observerUnactive();
+            if (block.nodeName === "DIV" && content === '') {
+                const p = this.document.createElement("p");
+                const br = this.document.createElement("br");
+                p.appendChild(br);
+                block.appendChild(p);
+                block  = p;
+                setSelection(p, 0);
```

Comment:
Are you using the `hint` mechanism to create missing paragraphs? ^^'

Reply:
Indeed, my idea is when you put the cursor we added the empty p otherwise there is no need for it